source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRContentStackView.mm

```diff
@@ -578,4 +578,12 @@ - (void)dealloc
     }
 }
 
+- (void)toggleVisibilityOfFirstView
+{
+    if ([_stackView.subviews count] and _stackView.subviews[0].hidden) {
+        _stackView.subviews[0].hidden = NO;
+        _stackView.subviews[0].hidden = YES;
+    }
```

Comment:
This seems...hacky. Why is it necessary to toggle it this way? Is there no alternative?

Reply:
It appears to be an Apple bug. iOS has a UIStackView which is similar to a Grid. If a view is added to the empty UIStackView for layout, The UIStackView will become invisible if the view is invisible. This is expected behavior since all of its subviews are invisible although the subviews count is 1. However, what is not expected is that when we subsequently add a visible view to the UIStackView, the UIStackView won't turn its visibility visible. We could have made UIStackView visible but it requires us to check all the subviews of the UIStackView to make sure that there is at least one subview that is visible. I feel that this is the job of the system. Although it looks hacky, I think letting the system decide the visibility looked better alternative.